Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,96 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render an `<picture>` element for responsive image support.
+public struct Picture: Component {
+    /// Type used to define an  Picture source, which points to an image file (or set of them) and associated media queries (and other paramaters).
+    public struct Source {
```

Comment:
Could you move this `struct` definition to an extension instead, to make the code a bit easier to read?

```swift
public extension Picture {
    struct Source {
        ...
    }
}
```

Reply:
Done. I understand why you feel it makes it more readable (and easier to follow). I included it in the parent `struct` as it is required, and so did not feel like an `extension` to me.